roles/config_drive/defaults/main.yml

```diff
@@ -0,0 +1,65 @@
+---
+# Copyright Red Hat, Inc.
+# All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may
+# not use this file except in compliance with the License. You may obtain
+# a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+
+
+# All variables intended for modification should be placed in this file.
+# All variables within this role should have a prefix of "cifmw_config_drive"
+cifmw_config_drive_basedir: >-
+  {{
+    cifmw_basedir | default(ansible_user_dir ~ '/ci-framework-data')
+  }}
+cifmw_config_drive_workdir: "{{ cifmw_config_drive_basedir }}/cifmw_config_drive"
+
+cifmw_config_drive_uuid: uuid-test
+cifmw_config_drive_instancedir: "{{ cifmw_config_drive_workdir }}/{{ cifmw_config_drive_uuid }}"
+cifmw_config_drive_iso_image: "{{ cifmw_config_drive_workdir }}/{{ cifmw_config_drive_uuid }}.iso"
+cifmw_config_drive_name: test
+cifmw_config_drive_hostname: test.example.com
+cifmw_config_drive_userdata:
+# user-data - example
```

Comment:
Is there a better place to put an example other than with comments?
Wouldn't block on this.

Reply:
There is an example in the README, I guess we can drop this. (If we merge before I have chance to update it I can do a follow up to remove examples from here.)

Reply:
+1 to removing it, I don't mind leaving the empty vars but the full example should live only in the README to prevent having multiple places to update in the future.

Reply:
Done